scripts/cleandb.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+# Load .env file if present
+set -a; source .env; set +a
+# abort on exception
+set =e
+
+# Get DB name from connection URI
+dbname=${POSTGRES_URL#*//*/}
+# Remove DB name from connection URI
+postgres_url=${POSTGRES_URL/$dbname}
+
+echo $dbname
+
+psql $postgres_url -c "DROP SCHEMA public CASCADE"
+psql $postgres_url -c "CREATE SCHEMA public"
+psql $postgres_url -c "DROP DATABASE IF EXISTS $dbname"
```

Comment:
why do we need to drop the schema if we can just drop the database? also, why do we need to drop the database *after* dropping the schema? i'm not sure how the `IF EXISTS` could ever be true here.

Reply:
Oh, thanks!  This was some lazy copy pasta on my part from `tests/server/reset-db.sh`.  It's been a while since I did a lot of SQL work.